powerline/lint/__init__.py

```diff
@@ -126,6 +126,12 @@ def load_json_config(config_file_path, load=load, open_file=open_file):
 				select=ext_theme_spec(),
 			),
 		).optional(),
+		wm=ext_spec().update(
+			local_themes=Spec().unknown_spec(
+				Spec().re('[A-Z0-9-]+'),
```

Comment:
`[A-Z0-9-]+` literally means “starts with a latin capital letter, digit or hyphen-minus” and looks like “contains latin capital letter, digit or hyphen-minus”. You are missing anchors: `^` and `$`.